paddle/scripts/travis/build_and_test.sh

```diff
@@ -1,7 +1,22 @@
 #!/bin/bash
 source ./common.sh
-cmake .. -DCMAKE_BUILD_TYPE=Debug -DWITH_GPU=OFF -DWITH_DOC=OFF -DWITH_TESTING=ON -DON_TRAVIS=ON
-make -j `nproc`
-env CTEST_OUTPUT_ON_FAILURE=1 make test ARGS="-j `nproc`"
+CMAKE_EXTRA=""
+if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
+  CMAKE_EXTRA="-DPYTHON_LIBRARY=/usr/local/Cellar/python/2.7.12_1/Frameworks/Python.framework/Versions/2.7/lib/python2.7/config/libpython2.7.dylib"
```

Comment:
I know this hard code setting is ugly, but it seems the only way make travis to find correct python. Because in travis mac os environment, two pythons are installed.